docs/index.md

```diff
@@ -8,13 +8,13 @@ id: home
   <div class="marketing-row">
     <div class="marketing-col">
       <h3>Declarative</h3>
-      <p>React makes it painless to update your views when your data changes. Just return the views to display for each possible state, and React handles the updates.</p>
+      <p>React lets you work on dynamic UIs as if they were static. Describe what your views look like in different states, and React will take care of updating them whenever the data changes. </p>
```

Comment:
“Return the views” sounded a little confusing because most people who know what “views” are come from traditional MVC paradigm where you don’t return them. So I tried to go for an alternative explanation instead.